linter/lookml_linter.py

```diff
@@ -20,10 +20,13 @@ def run(self) -> None:
             file_data = self.data[filename]
             views = file_data.get('views', [])
             explores = file_data.get('explores', [])
+            includes = file_data.get('includes', [])
             self._errors.append({
                 'filename': filename,
                 'messages': []
             })
+            for i in includes:
+                self.__lint_object(i, 'include')
```

Comment:
lkml parses includes as a list of strings, e.g.:

```
['some_include.view', 'another_include.view']
```

You can run the project locally and confirm this by doing:

```
python -m pipenv install
python -m pipenv shell
python -m linter.main
```

So doing `self.__lint_object(i, 'include')` will break on line 68 (`object["name"]`), since `i` is not an object and doesn't have a name property.  This is why you would need to do something like:

```
self.__lint_object({
    "name": i
}, 'include'})
```